conda/lock.py

```diff
@@ -20,13 +20,31 @@
 import logging
 import os
 import time
+from glob import glob
```

Comment:
In general, prefer `from MODULE import FUNCTION` instead of using `MODULE.FUNCTION` in-line.  I haven't refactored the whole file in this PR to keep it clean, but this should be the general pattern, especially for any new code or code lines that we're touching and changing.